api/src/wfl/module/covid.clj

```diff
@@ -127,34 +195,12 @@
         (->> (jdbc/execute! tx)))
     items))
 
-(defn ^:private patch-workload [tx {:keys [id]} colls]
-  (jdbc/update! tx :workload colls ["id = ?" id]))
-
-(defn ^:private patch-continuous-workload
-  "Use `tx` to update the colls of the `_workload`."
-  [tx {:keys [items] :as _workload} colls]
-  (jdbc/update! tx continuous-workload-table-name colls
-                ["id = ?" (util/parse-int items)]))
-
-(defn ^:private add-workload-record
-  "Use `tx` to create a workload `record` for `request` and return the id of the
-   new workload."
+(defn create-covid-workload
+  "Verify the `request` and create a workload"
   [tx request]
-  (letfn [(combine-labels [labels]
```

Comment:
Just want to make sure that you meant to delete this?

Reply:
Yeah - that was a merge conflict issue. I think it's correct.